libraries/abstractions/transport/secure_sockets/transport_secure_sockets.c

```diff
@@ -210,9 +214,9 @@ static int32_t tlsSetup( const SocketsConfig_t * pSocketsConfig,
 {
     int32_t secureSocketStatus = SOCKETS_ERROR_NONE;
 
-    configASSERT( tcpSocket != SOCKETS_INVALID_SOCKET );
-    configASSERT( pSocketsConfig != NULL );
-    configASSERT( pHostName != NULL );
+    configASSERT( ( tcpSocket != SOCKETS_INVALID_SOCKET ) ? pdTRUE : pdFALSE );
```

Comment:
I don't understand why we need this ternary here. In the C SDK, we disable asserts before checking MISRA. Can the same be done here? Otherwise, this looks awkward to use every time we use a `configASSERT`

Reply:
If so, I can disable it because I run MISRA with ```configASSERT```. Thanks for your comment.